src/resources/Pipelines/ResultRankings/ResultRankingsInterfaces.ts

```diff
@@ -102,4 +125,6 @@ export interface ListResultRankingParams {
     page?: number;
     perPage?: number;
     organizationId?: string;
+    associatedGroups?: Array<string | null>;
+    kind?: string;
```

Comment:
I could add the explicit list of supported types here if needed

> "featured_results" | "ranking_expressions"

Reply:
We already have it [here](https://github.com/coveo/platform-client/blob/2c5d1ec24e2db63e1227268e9c2780b0c35d950d/src/resources/Enums.ts#L646), I think it could be an easy win.

Reply:
I'll reuse this one then!